### PR TITLE
[Core] Console queries

### PR DESCRIPTION
**Pull request by Jkat**
Can someone test if this works?

### CODE REVIEW

**Comment by driusan** [comment]
Travis says no.

File: php/libraries/Database.class.inc [hunk 1]
```diff
@@ -974,9 +974,11 @@ class Database
         if ($params) {
             $query = str_replace(array_keys($params), array_values($params), $query);
         }
-        print "$this->_databaseName:"
+        $query = json_encode($query);
+        print "console.log('"
```
Comment (MounaSafiHarab) [inline php/libraries/Database.class.inc]: @alisterdev @driusan  say use: console.info instead so that Travis does not fail

**Comment by Jkat** [comment]
TRAVIS SAID YES!!!!!! TRAVIS SAID YES!!!!!! TRAVIS SAID YES!!!!!! TRAVIS SAID YES!!!!!! TRAVIS SAID YES!!!!!! TRAVIS SAID YES!!!!!! TRAVIS SAID YES!!!!!! TRAVIS SAID YES!!!!!! TRAVIS SAID YES!!!!!! TRAVIS SAID YES!!!!!! TRAVIS SAID YES!!!!!! TRAVIS SAID YES!!!!!! TRAVIS SAID YES!!!!!! TRAVIS SAID YES!!!!!! TRAVIS SAID YES!!!!!! TRAVIS SAID YES!!!!!! TRAVIS SAID YES!!!!!! TRAVIS SAID YES!!!!!! TRAVIS SAID YES!!!!!! TRAVIS SAID YES!!!!!! TRAVIS SAID YES!!!!!! AHHHHH!!!!!!!!!!! HALLELUJAH :pray:

File: php/libraries/Database.class.inc [hunk 2]
```diff
@@ -974,9 +974,11 @@ class Database
         if ($params) {
             $query = str_replace(array_keys($params), array_values($params), $query);
         }
-        print "$this->_databaseName:"
-            . date("j-M-Y G:i:s", time())
-            .": $query<br>\n";
+        $query = json_encode($query);
+        print "console.info('"
```
Comment (alisterdev) [inline php/libraries/Database.class.inc]: It seems to print out the output as a string and append it to body for some reason
image

This breaks the page and code never gets to execute your part of code

File: smarty/templates/main.tpl [hunk 3]
```diff
@@ -206,30 +208,12 @@
         <div id="page-content-wrapper">
             {/if}
             {if $dynamictabs eq "dynamictabs"}
-                {if $console}
```
Comment (alisterdev) [inline smarty/templates/main.tpl]: Can it be the case that somebody prefers seeing queries directly on the page, so maybe we need an option to keep 1 or both?

**Comment by MounaSafiHarab** [comment]
@gluneau is working on putting these messages into the error_log instead; here: https://github.com/aces/Loris/pull/2572

**Pull request closed without merging**